### ISSUE TITLE
Latest Build Failing

### ISSUE THREAD

**Issue by shakna-israel** [shakna-israel/NecromancersApprentice#5]
```
Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/necromancersapprentice/envs/latest/bin/mkdocs", line 9, in <module>
    load_entry_point('mkdocs==0.11.1', 'console_scripts', 'mkdocs')()
  File "/home/docs/checkouts/readthedocs.org/user_builds/necromancersapprentice/envs/latest/local/lib/python2.7/site-packages/mkdocs/main.py", line 60, in run_main
    main(cmd, args=sys.argv[2:], options=dict(opts))
  File "/home/docs/checkouts/readthedocs.org/user_builds/necromancersapprentice/envs/latest/local/lib/python2.7/site-packages/mkdocs/main.py", line 36, in main
    build(config, clean_site_dir=clean_site_dir)
  File "/home/docs/checkouts/readthedocs.org/user_builds/necromancersapprentice/envs/latest/local/lib/python2.7/site-packages/mkdocs/build.py", line 266, in build
    build_pages(config)
  File "/home/docs/checkouts/readthedocs.org/user_builds/necromancersapprentice/envs/latest/local/lib/python2.7/site-packages/mkdocs/build.py", line 208, in build_pages
    input_content = input_content.decode('utf-8')
  File "/home/docs/checkouts/readthedocs.org/user_builds/necromancersapprentice/envs/latest/lib/python2.7/encodings/utf_8.py", line 16, in decode
    return codecs.utf_8_decode(input, errors, True)
UnicodeDecodeError: 'utf8' codec can't decode byte 0x85 in position 2067: invalid start byte
```

Commit: f5ce50a4a1235a6e65ddb9752180cf7e70c9a876

**Comment by shakna-israel** [comment]
That would be Windows... ANSI instead of UTF-8 format.

**Comment by shakna-israel** [comment]
Fixed as of 6fb0fbb7450dfd9c7ffd48a313da9cc78d99db51
